src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -2056,9 +2056,19 @@ in storage.js plus the media query with (min-width: 701px)
 }
 
 @media print {
-	nav.sub, .content .out-of-band {
+	nav.sidebar, nav.sub, .content .out-of-band, a.srclink, #copy-path,
+	details.rustdoc-toggle[open] > summary::before, details.rustdoc-toggle > summary::before,
```

Comment:
Shouldn't `details.rustdoc-toggle > summary::before,` be enough? I think it covers `[open]` as well.

Reply:
Intriguing. Well, since you tested it I guess it's fine.